desk/app/reel.hoon

```diff
@@ -93,6 +109,9 @@
   ?>  =(our.bowl src.bowl)
   ?+  path  (on-watch:def path)
     [%bites ~]  `this
+      [%token-link @ @ ~]
+    :_  this
```

Comment:
One worry with this poke-based approach is that if you refresh many times and the group host is never online, you could end up with many outstanding pokes, since they don't get backpressure.

One way to solve this would be to keep a `(set [ship cord])` of outstanding pokes.  Here, if already in the set, don't send the poke again; else, add to the set.  Remove from the set in on-agent's poke-ack handling (regardless if positive or negative).

Same applies to %grouper